org.wso2.codequalitymatrices/src/main/java/com/wso2/code/quality/matrices/model/Repository.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.wso2.code.quality.matrices.model;
+
+/**
+ * Pojo class used for parsing JSON response received from github REST API
+ */
+public class Repository {
+    private String full_name;
```

Comment:
camel case notation cannot be used in here as we need to use the same field name in the JSON for mapping with GSON. For example in [https://api.github.com/repos/wso2/wso2-axis2-transports/commits/e3c3457149b109178d510aac965d5a85cc465aa0](this) "html_url", "comments_url" fields are named with having " _"